app/assets/stylesheets/dashboard.scss

```diff
@@ -328,7 +328,7 @@
   }
   .no-articles {
     text-align: center;
-    background: white;
+    background: var(--theme-container-accent-background, $light-gray);
```

Comment:
Also not sure if they want to change the default style, I would suggest to leave it `white`.